tests/test_cookiecutter_django_app.py

```diff
@@ -175,3 +175,11 @@ def test_quality(options_baked):
         sh.doc8("docs", ignore_path="docs/_build")
     except sh.ErrorReturnCode as exc:
         pytest.fail(str(exc))
+
+
+def test_tests(options_baked):
+    """Make sure the tox default tests work"""
+    try:
+        run_in_virtualenv('pip install -r requirements/ci.txt; tox')
```

Comment:
This tries to match the GitHub check as closely as possible -- which is part of why I rearranged the tox default envlist so we can just run `tox` to get the pytest runs. (Our repos seem split on which approach they take for `envlist`.)

Reply:
Do we need a small ADR on approaches and why we are taking one over the other?

Reply:
I'm not sure if it's worth doing. Maybe if we decide to apply this to all the templates?